pkg/resourcecreator/testdata/leader_election_simple.yaml

```diff
@@ -0,0 +1,79 @@
+config:
+  description: leader election container injection, using simple elector
+
+resourceoptions:
+  LeaderElection:
+    Image: "not-a-google-containers-image:latest"
+
+input:
+  kind: Application
+  apiVersion: v1alpha1
+  metadata:
+    name: myapplication
+    namespace: mynamespace
+    uid: "123456"
+    labels:
+      team: myteam
+  spec:
+    leaderElection: true
+
+tests:
+  - apiVersion: rbac.authorization.k8s.io/v1
+    kind: ClusterRole
+    name: myapplication
```

Comment:
Tenker litt fremover, vi går ikke i beina på noe vi kan ønske lager senere ved å lage en clusterrole(binding) med samme navn som appen?

Reply:
Kanskje lurt med en prefix av et slag ja

Reply:
Dette er egentlig bare en videreføring av slik det allerede var, forskjellen er bare at det nå er en ClusterRole og ClusterRoleBinding i steden for Role og RoleBinding. Men siden Cluster* variantene ikke er begrenset til namespace gir det kanskje mening å bake namespace inn i navnet på et vis? Da blir det fort problemer med lengden på navnet.. :thinking:

Reply:
Et bedre spørsmål er hvorfor det nå trengte å være ClusterRole.. jeg husker ikke hvorfor det ble sånn. :thinking:

Reply:
Det er jo bare tull, kan være vanlig Role/Rolebinding. Kan selvfølgelig legge på "elector-" først eller noe, for å gi dem mer beskrivende navn.

Reply:
Endre til vanlig Role/RoleBinding, og lagt på prefix på navnet på disse.